infra/Pos/Infra/Configuration.hs

```diff
@@ -1,10 +1,9 @@
 {-# LANGUAGE Rank2Types #-}
 module Pos.Infra.Configuration
```

Comment:
Should move this file to `infra/Pos/Ntp/Configuration.hs`

Reply:
Let's also move `infra/Pos/NtpCheck.hs` to `infra/Pos/Ntp/NtpCheck.hs`.

Reply:
Or even better, just `Pos.Ntp.Check`